src/locale/layout/orgHeader/en.yml

```diff
@@ -0,0 +1,2 @@
+editLink: Edit Page
+unfollowButton: Unfollow
```

Comment:
I'm not sure whether I think this is a good place for this file. Is `orgHeader` really a layout? If not, maybe it's here because it's a part of `OrgLayout`? Well, then why have a separate set of YAML files at all? Another solution would be to just add these two strings to the `org` set of files, like this:

```yml
# src/locale/layout/org/en.yml
header:
  editLink: Edit page
  unfollowButton: Unfollow
tabs:
  home: Home
  comingUp: Coming up
  campaigns: Campaigns
```